lesson02/main.tf

```diff
@@ -3,10 +3,20 @@ provider "aws" {
   region = var.aws_region
 }
 
-terraform {
-  backend "s3" {
-    bucket = "wizeline-academy-terraform"
-    region = "us-east-2"
+# If you prefer, you can store the Terraform state in S3
+# https://www.terraform.io/docs/backends/types/s3.html
+# terraform {
+#   backend "s3" {
+#     region = "us-east-2"
+#   }
+# }
+
+#--------------------------------------------------------------
+# default VPC
+# https://www.terraform.io/docs/providers/aws/r/default_vpc.html
+#--------------------------------------------------------------
+resource "aws_default_vpc" "default" {
+  tags = {
+    Name = "Default VPC"
```

Comment:
NBD: would be nice having also a tag mentioning the resource was created using Terraform
e.g. `terraform = "true"`

Reply:
Should we include that in all lessons?